ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/VariableProcessor.kt

```diff
@@ -0,0 +1,51 @@
+package com.revenuecat.purchases.ui.revenuecatui.data.processed
+
+import android.content.Context
+import com.revenuecat.purchases.Package
+import com.revenuecat.purchases.ui.revenuecatui.helpers.Logger
+import java.util.Locale
+
+internal object VariableProcessor {
+    private val REGEX = Regex("\\{\\{\\s[a-zA-Z0-9_]+\\s\\}\\}")
+
+    fun processVariables(
```

Comment:
This could really use tests too (see `VariablesTests`).

Reply:
Added in https://github.com/RevenueCat/purchases-android/pull/1251/commits/d70f707a573e02f1a82f2b026d7cc75ef7e94886